src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -133,6 +134,19 @@ public Boolean isKnownParticipant(
         return chronicleService.isKnownParticipant( studyId, participantId );
     }
 
+    @Override
+    public Void deleteParticipantAndAllNeighbors(
```

Comment:
Use `public void` and remove the `return null`.

Reply:
ah, that would conflict with the definition in chronicle-api

Reply:
`chronicle-api` has been updated. Please make the change.

https://github.com/openlattice/chronicle-api/pull/23

Reply:
Updated